be/src/storage/persistent_index.cpp

```diff
@@ -2,19 +2,66 @@
 
 #include "storage/persistent_index.h"
 
+#include <string.h>
```

Comment:
1. `<cstring>` ? 
2. remove the line separator?

Reply:
fixed